Topics/statistics-fundamentals.md

```diff
@@ -0,0 +1,167 @@
+
+**Keep it as a reference!**
+
+#### Check your knowledge
+
+Test your knowledge with some of [these exercises](https://www0.gsb.columbia.edu/faculty/pglasserman/B6014/exercises.pdf).
+
+Anyway, you can find a ton of them online, and remember, that the more you practice, the better you will get!
+
+
+
+### Probability Theory
+
+Probability theory governs every daily event, from microscopic to macroscopic, even if we hardly realize it!
+
+But you may have thought about the odds of winning, for example, the game of dice, or when you estimate the probability of having successfully passed an exam!
+
+Probability theory is so interconnected with statistical and mathematical concepts that it _is difficult to explain why it is fundamental_: simply consider it as the basic building blocks of more advanced statistical concepts based on it.
+
+Not to mention that knowledge of probability theory can be very useful in everyday life since it can suggest to us how to behave on certain occasions, to make the best choices, or involving the least risk of failure.
+
+Fortunately, you can have the (perhaps) best course on probability theory available to date, offered as usual by MIT. 
+
+It's good to have a first-class education from home for free, huh? 
+
+What an incredible time to be alive!
+
+Take the course at:
+
+[**Introduction to Probability - Video Lectures**](https://www.youtube.com/watch?v=1uW3qMFA9Ho&list=PLUl4u3cNGP60hI9ATjSFgLZpbNJ7myAg6)
+
+In addition, keep this _free complete book_ as a reference:
+
+[**Introduction to Probability - Book**](https://www.dartmouth.edu/~chance/teaching_aids/books_articles/probability_book/amsbook.mac.pdf)
+
+#### Check your knowledge
+
+Test your knowledge with some of [these exercises](https://www.math.kth.se/matstat/gru/sf1901/TCOMK/exercises.pdf).
+
+Anyway, you can find a ton of them online, and remember, that the more you practice, the better you will get!
+
+### Bayesian Statistics
+
+This last topic is not fundamental for a first pass study, but it's highly recommended since some of Machine Learning algorithms are based on it or leverage some concepts from Bayesian Theory.
+
+The Bayesian approach is often distinguished from the "classical" approach, which you have studied in the [Descriptive Statistics](#Descriptive-Statistics) section, and which is called "frequentist" when compared to the "Bayesian" approach.
+
+The Bayesian approach to statistics is based upon the famous [Bayes Theorem](https://en.wikipedia.org/wiki/Bayes%27_theorem), which states that the probability of an event when prior evidence is given and that might be related to that event.
+
+For example, if cancer is related to age, then, using Bayes’ theorem, a person’s age can be used to more precisely or accurately assess the probability that they have cancer, in contrast to the assessment of the probability of cancer made without knowledge of the person's age.
```

Comment:
I have a nice article to share about this, and maybe it can be added here: https://lavanya.ai/2019/05/16/bayes-theorem/ also this [kaggle kernel](
    - https://www.kaggle.com/chandan3231/naive-bayes-probabilistic-ml-titanic-survival
    - ) explain the concept with a simple example and diagram.

Reply:
thanks the second is a very good link. I put it